subsys/fs/fs.c

```diff
@@ -149,7 +149,7 @@ int fs_open(struct fs_file_t *zfp, const char *file_name, fs_mode_t flags)
 
 	rc = fs_get_mnt_point(&mp, file_name, NULL);
 	if (rc < 0) {
-		LOG_ERR("%s:mount point not found!!", __func__);
+		LOG_ERR("%s:mount point not found!!", log_strdup(__func__));
```

Comment:
It is better to remove all `__func__` because logging subsystem offers an option to prepend messages with function name (LOG_FUNC_NAME_PREFIX_ERR).

Reply:
Have updated the PR with your suggestion.